grpc-interface-support/src/cpp.py

```diff
@@ -346,28 +360,57 @@ def __create_or_update_service_header(self) -> None:
         service_header_file_path = os.path.join(
             app_source_dir, service_header_file_name
         )
-
-        user_defined_code: List[str] = []
-        if os.path.exists(service_header_file_path):
-            # there is a previous version of the header
-            # extract the user-defined code
-
-            user_defined_code = capture_area_in_file(
-                open(service_header_file_path, encoding="utf-8"),
-                "// <user-defined>",
-                "// </user-defined>",
-            )
-
         variables = self.__get_template_variables()
         variables["service_header_code"] = "\n".join(header_stub_code)
-        variables["service_header_user_code"] = "\n".join(user_defined_code)
 
         copy_templates(
             get_template_dir(),
             app_source_dir,
             [CopySpec("ServiceImpl.h", service_header_file_name)],
             variables,
         )
+        
+    def __update_service_header(self) -> None:
+        header_generated_code = GrpcCodeExtractor(
+            self.__proto_file_handle, self.__package_directory_path
+        ).get_header_stub_code(self.__get_include_dir())
+
+        header_generated_code = self.__transform_header_stub_code(header_stub_code)
```

Comment:
Probably a copy paste error? I get the error that `header_stub_code` is not defined:

```
File "/home/vscode/.velocitas/packages/devenv-devcontainer-setup/@update_autogenerated_code_only/grpc-interface-support/src/cpp.py", line 378, in __update_service_header
    header_generated_code = self.__transform_header_stub_code(header_stub_code)
NameError: name 'header_stub_code' is not defined
```